src/core/a-lib.c

```diff
@@ -313,7 +297,7 @@ extern int Do_Callback(REBSER *obj, u32 name, RXIARG *args, RXIARG *result);
 
 #ifdef DUMP_INIT_SCRIPT
 	f = _open("host-boot.r", _O_CREAT | _O_RDWR, _S_IREAD | _S_IWRITE );
-	_write(f, STR_HEAD(text), LEN_BYTES(STR_HEAD(text)));
+	_write(f, STR_HEAD(text), strlen(STR_HEAD(text)));
```

Comment:
The REBYTE-based rewriting of C memory and string routines that took char * wasn't helping make the code more accessible (and it they weren't type correct).  See reb-c.h for descriptions of the new plane.